README.md

```diff
@@ -9,31 +9,24 @@ If you're new to the project, welcome! Please feel free to ask questions [here](
 
 This project provides representative performance measures across a wide field of web application frameworks. With much help from the community, coverage is quite broad and we are happy to broaden it further with contributions. The project presently includes frameworks on many languages including `Go`, `Python`, `Java`, `Ruby`, `PHP`, `C#`, `Clojure`, `Groovy`, `Dart`, `JavaScript`, `Erlang`, `Haskell`, `Scala`, `Perl`, `Lua`, `C`, and others.  The current tests exercise plaintext responses, JSON seralization, database reads and writes via the object-relational mapper (ORM), collections, sorting, server-side templates, and XSS counter-measures. Future tests will exercise other components and greater computation.
 
-[Read more and see the results of our tests on Amazon EC2 and physical hardware](http://www.techempower.com/benchmarks/). 
-For descriptions of the test types that we run, see the 
+[Read more and see the results of our tests on cloud and physical hardware](http://www.techempower.com/benchmarks/). For descriptions of the test types that we run, see the 
 [test requirements section](https://frameworkbenchmarks.readthedocs.org/en/latest/Project-Information/Framework-Tests/).
 
 If you find yourself in a directory or file that you're not sure what the purpose is, checkout our [file structure](http://frameworkbenchmarks.readthedocs.org/en/latest/Codebase/#file-structure) in our documenation, which will briefly explain the use of relevant directories and files.
 
 ## Quick Start Guide
 
-Get started developing quickly by utilizing vagrant with TFB. [Git](https://git-scm.com), 
-[Virtualbox](https://www.virtualbox.org/) and [vagrant](https://www.vagrantup.com/) are 
-required.
+To get started developing you'll need to install [docker](https://docs.docker.com/install/).
```

Comment:
Maybe something here about "or optionally install VirtualBox and Vagrant" and link to that header in this README.md